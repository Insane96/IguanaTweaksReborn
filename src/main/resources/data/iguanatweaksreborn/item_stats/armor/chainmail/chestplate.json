{
	"item": "minecraft:chainmail_chestplate",
	"durability": 64,
	"armor": 5,
	"toughness": 1
}