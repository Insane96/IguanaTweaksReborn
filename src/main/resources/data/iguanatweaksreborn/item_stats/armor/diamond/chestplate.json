{
	"item": "minecraft:diamond_chestplate",
	"durability": 240,
	"armor": 6,
	"toughness": 1,
	"knockback_resistance": 0.03
}