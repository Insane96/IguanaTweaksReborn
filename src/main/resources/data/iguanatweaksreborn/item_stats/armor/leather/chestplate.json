{
	"item": "minecraft:leather_chestplate",
	"durability": 96,
	"armor": 3,
	"toughness": 1
}