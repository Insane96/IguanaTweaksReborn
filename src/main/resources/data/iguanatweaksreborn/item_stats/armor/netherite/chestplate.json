{
	"item": "minecraft:netherite_chestplate",
	"durability": 224,
	"armor": 7,
	"toughness": 2,
	"knockback_resistance": 0.05
}