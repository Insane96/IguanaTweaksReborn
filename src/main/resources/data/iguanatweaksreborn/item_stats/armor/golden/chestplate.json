{
	"item": "minecraft:golden_chestplate",
	"durability": 96,
	"armor": 5,
	"toughness": 1
}